### ISSUE TITLE
Subtitles and content list out of date

### ISSUE THREAD

**Issue by prrao87** [superlinked/VectorHub#68]
- [ ] Typo
- [x] Content out of date

![image](https://github.com/superlinked/VectorHub/assets/35005448/7ecf1964-94e6-480a-9e22-4987d55c2c85)

The articles in the home page aren't in sync with those in the side bar (there are only 3 shown in the main page but 6 in the side-bar), and the section titles also aren't in sync (the side bar says 'blog" but the home page says "vector retrieval use-cases" - just wanted to point that out!

**Comment by ClaireSuperlinked** [comment]
Thanks for this - you're right! So we set up the taxonomy with use cases and then all blog articles go into different use cases (if you click blog then you see them all listed).


There is an open project to fix the Taxonomy of VectorHub as we know it sucks, and when we do, we will change the homepage to reflect the updated taxonomy making it easier to navigate. 

Initially everything was designed around a specific structure which we realised wouldn't scale. We didn't want to duplicate effort by changing everything twice in quick succession but this is on the roadmap to resolve in Jan.
